azurerm/resource_arm_sql_server.go

```diff
@@ -74,6 +75,33 @@ func resourceArmSqlServer() *schema.Resource {
 				Computed: true,
 			},
 
+			"identity": {
```

Comment:
could we [add this new block to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/sql_server.html.markdown#L42-L54)? there's [an example of how that looks for another resource here](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/app_service.html.markdown#L94))

Reply:
Added documentation, including updated data source.